src/dbos-runtime/config.ts

```diff
@@ -113,15 +114,18 @@ export function constructPoolConfig(configFile: ConfigFile, useProxy: boolean =
   }
 
   // Details on Postgres SSL/TLS modes: https://www.postgresql.org/docs/current/libpq-ssl.html#LIBPQ-SSL-PROTECTION
-  if (configFile.database.ssl_ca) {
+  if (configFile.database.ssl === false) {
+    // If SSL is set to false, do not use TLS
+    poolConfig.ssl = false
+  } else if (configFile.database.ssl_ca) {
     // If an SSL certificate is provided, connect to Postgres using TLS and verify the server certificate. (equivalent to verify-full)
     poolConfig.ssl = { ca: [readFileSync(configFile.database.ssl_ca)], rejectUnauthorized: true };
-  } else if (poolConfig.host != "localhost" && poolConfig.host != "127.0.0.1") {
+  } else if (configFile.database.ssl !== true && (poolConfig.host === "localhost" || poolConfig.host === "127.0.0.1")) {
```

Comment:
`configFile.database.ssl !== true` will always be true? 🤔

Reply:
Agree this should be a separate `if (...)`

Reply:
But can we just disable SSL if it's not defined? We always guarantee SSL is set to `true` on the cloud. This logic is still complicated.